ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -1,4 +1,232 @@
+        name: longlived-aws-cluster-credentials
+    serviceAccountName: ci-operator
+    volumes:
+    - configMap:
+        name: prow-job-longlived-test
+      name: job-definition
+    - name: ci-credentials
+      secret:
+        secretName: cluster-secrets-aws
+    - name: pull-secret
+      secret:
+        secretName: ci-pull-credentials
+    - name: longlived-aws-cluster-credentials
+      secret:
+        secretName: longlived-aws-cluster-credentials
+- agent: kubernetes
+  cron: 0 12 * * 1-5
+  decorate: true
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: longlived-aws-cluster-e2e-4.1
+  spec:
+    containers:
+    - command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        ci-operator --artifact-dir=$(ARTIFACTS) \
+          --secret-dir=/usr/local/pull-secret \
+          --secret-dir=/usr/local/longlived-aws-cluster-credentials \
+          --template=/usr/local/longlived-test/longlived-test.yaml \
+          --input-hash=$(BUILD_ID) --input-hash=$(JOB_NAME) \
+          --target=longlived-test
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: registry.svc.ci.openshift.org/ocp/release:4.1-ci
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              requests:
+                cpu: 10m
+          tests:
+          - as: longlived-$(CLUSTER_TYPE)-e2e
+            commands: TEST_SUITE=openshift/conformance/parallel run-tests
+            openshift_installer:
+              cluster_profile: "$(CLUSTER_TYPE)"
+              upgrade: false
+      - name: JOB_NAME_SAFE
+        value: longlived-aws-e2e
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/longlived-test
+        name: job-definition
+        subPath: longlived-test.yaml
+      - mountPath: /usr/local/pull-secret
+        name: pull-secret
+      - mountPath: /usr/local/longlived-aws-cluster-credentials
+        name: longlived-aws-cluster-credentials
+    serviceAccountName: ci-operator
+    volumes:
+    - configMap:
+        name: prow-job-longlived-test
+      name: job-definition
+    - name: pull-secret
+      secret:
+        secretName: ci-pull-credentials
+    - name: longlived-aws-cluster-credentials
+      secret:
+        secretName: longlived-aws-cluster-credentials
+- agent: kubernetes
+  cron: 0 9 * * 1-5
+  decorate: true
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: longlived-aws-cluster-maintenance-4.1
+  spec:
+    containers:
+    - command:
+      - /bin/bash
+      - -c
+      - "#!/bin/bash\nset -euo pipefail\n\nexport PATH=$PATH:/tmp/bin\nmkdir /tmp/bin\ncurl
```

Comment:
Fix this formatting

Reply:
I can't. Make jobs does this.

Reply:
context: https://github.com/openshift/release/commit/ef3cd1d0c5a65fc78dc9b4a4d07d536d983554f1#r35370176

(you've argued with @stevekuznetsov about this before, it appears to have gone unresolved)

Reply:
Make jobs is a turd if it breaks human entered text.  Can you open a card for DPTP?  Needs to get fixed or disabled for these jobs.

Reply:
submitted a ticket to openshift-ci-requsts

Reply:
https://jira.coreos.com/browse/DPTP-658